modules/dreamview/backend/simulation_world/simulation_world_service.cc

```diff
@@ -955,13 +956,33 @@ void SimulationWorldService::UpdatePlanningData(const PlanningData &data) {
   }
 
   // Update planning signal
-  // TODO(zhangqian): check with planning to decide a reasonable signal
   world_.clear_traffic_signal();
   if (data.has_signal_light() &&
       data.signal_light().signal_size() > 0) {
+    TrafficLight::Color current_signal = TrafficLight::UNKNOWN;
+    int green_light_count = 0;
+
+    for (auto &signal : data.signal_light().signal()) {
```

Comment:
If planning is just looking at perception signals, should we implement this logic upon receiving perception traffic light?

Reply:
Planning is looking at traffic light overlapped with its reference line, and then get these traffic lights' color from perception.